lesson7/thingspeak_cpu_loop.py

```diff
@@ -2,12 +2,19 @@
 from time import localtime, strftime
 import psutil
 import time
+from pickle import dump as pickle_dump
+from pickle import load as pickle_load
```

Comment:
Pickle data type used to store credentials. Pickle is a binary storage version of JSON. Please note not-human-readable does not mean secure.